test/server/cards/01_SOR/leaders/CassianAndorDedicatedToTheRebellion.spec.ts

```diff
@@ -0,0 +1,95 @@
+describe('Cassian Andor, Dedicated to the Rebellion', function() {
+    integration(function(contextRef) {
+        describe('Cassian Andor\'s leader ability', function() {
+            beforeEach(function() {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        deck: ['k2so#cassians-counterpart'],
+                        groundArena: ['yoda#old-master'],
+                        spaceArena: ['green-squadron-awing'],
+                        leader: { card: 'cassian-andor#dedicated-to-the-rebellion', deployed: false },
+                        base: { card: 'dagobah-swamp', damage: 5 },
```

Comment:
What's the reason to declare damage on the bases in these tests? Typically we try not to add any test setup that isn't explicitly needed for the relevant case(s). Generally looks like a lot of stuff in the setup isn't used.

Reply:
Cleaned this up - removed as much unnecessary setup as possible